notifiers/notifier.go

```diff
@@ -0,0 +1,83 @@
+package notifiers
+
+import (
+	"encoding/json"
+	"finala/api/storage"
```

Comment:
Do not use finala storage in your notifier command. 
The notifier should be a charge of the response parsing